internal/cmd/configure/list.go

```diff
@@ -0,0 +1,22 @@
+package configure
+
+import (
+	"github.com/saucelabs/saucectl/internal/credentials"
+	"github.com/spf13/cobra"
+)
+
+func ListCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use: "list",
+		Aliases: []string{
+			"ls",
+		},
+		Short: "Showing the current credentials and resources",
```

Comment:
Not sure what `resources` meant here, so 🔪 :
```suggestion
		Short: "Shows the current credentials and their origin.",
```